ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -592,84 +86,14 @@
 			buildActionMask = 2147483647;
 			files = (
 				ED297163215061F000B7C4FE /* JavaScriptCore.framework in Frameworks */,
```

Comment:
Kind of puzzled why this line remains here. But, it sure is there in the template project! So be it, then.

Reply:
I'm about 80% sure I saw someone commenting that it was necessary to preserve this even after CocoaPods; the other 20% is that I guess I might have been thinking of something like [this](https://stackoverflow.com/a/54844185/6792075), which was just someone pointing out its addition in v0.57 or v0.58 (we're already on v0.59).